gsrs-fda-substance-extension/src/main/java/fda/gsrs/substance/exporters/ExcelSubstanceRelatedClinicalTrialsEuropeExporterFactory.java

```diff
@@ -0,0 +1,44 @@
+package fda.gsrs.substance.exporters;
+
+import gov.hhs.gsrs.clinicaltrial.europe.api.ClinicalTrialsEuropeApi;
+import gsrs.springUtils.AutowireHelper;
+import ix.ginas.exporters.Exporter;
+import ix.ginas.exporters.ExporterFactory;
+import ix.ginas.exporters.OutputFormat;
+import ix.ginas.models.v1.Substance;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Collections;
+import java.util.Set;
+
+public class ExcelSubstanceRelatedClinicalTrialsEuropeExporterFactory implements ExporterFactory{
+
+    @Autowired
+    public ClinicalTrialsEuropeApi clinicalTrialsEuropeApi;
+
+    OutputFormat format = new OutputFormat("cteuxlsx", "Spreadsheet File");
```

Comment:
I think the final export should still be the right extension (.xlsx) otherwise it'll behave strangely. "cteu.xlsx" should work.

Reply:
May be okay now that I'm seeing this isn't actually used in the real export ...